code/controllers/configuration.dm

```diff
@@ -239,6 +244,18 @@
 				if("use_age_restriction_for_antags")
 					config.use_age_restriction_for_antags = 1
 
+				if("use_exp_tracking")
+					config.use_exp_tracking = 1
+
+				if("use_exp_restrictions_heads")
+					config.use_exp_restrictions_heads = 1
+
+				if("use_exp_restrictions_heads_hours")
+					config.use_exp_restrictions_heads_hours = value

```

Comment:
Also, since this is used as minutes everywhere else, you should probably just convert this straight to minutes. The config option can stay hours, but you'll want to rename this var so it reflects that it's minutes.